devtools/prod-envs/qcarchive-worker-openff-psi4-basis-set-update.yaml

```diff
@@ -0,0 +1,24 @@
+name: qcarchive-worker-openff-psi4
+channels:
+  - conda-forge/label/libint_dev
+  - conda-forge
+dependencies:
+  - python =3.10
+  - pip
+  - qcfractalcompute>=0.61
+  - qcengine
+  - qcelemental
+
+  # QM calculations
+  - psi4
+  - pyddx 
+  - dftd3-python
+  - gcp-correction
+  - scipy
+  - dftd4-python
+    
+  # procedures
+  - geometric
+
+  - pip:
+    - basis_set_exchange
```

Comment:
There's a conda package for this (of the same name)

Reply:
(not blocking) The conda package seems to be kept up to date, but it's not referenced in the [docs](https://molssi-bse.github.io/basis_set_exchange/usage.html) or [repo README](https://github.com/MolSSI-BSE/basis_set_exchange), both of which point to the package on pypi. It's likely fine, just something to keep an eye on if it stops being updated.